To-do list/Models/Alert.swift

```diff
@@ -20,4 +20,17 @@ class Alert {
         alert.addAction(okAction)
         vc.present(alert,animated: true)
     }
+    
+    static func showBasicWithTimer(title: String,
```

Comment:
So, it is a bad pattern to define a class `Alert` which is non-UI related, and pass to it's function any UI class (UIView, UIViewController etc). 
Instead, you should extend `UIViewController` class and add these function to show alerts.